Java/corecrypto/src/main/java/com/breadwallet/corecrypto/System.java

```diff
@@ -1660,26 +1661,31 @@ private static void btcGetTransactions(Cookie context, BRCryptoWalletManager cor
                                 new CompletionHandler<List<Transaction>, QueryError>() {
                                     @Override
                                     public void handleData(List<Transaction> transactions) {
+                                        boolean success = false;
                                         Log.log(Level.FINE, "BRCryptoCWMBtcGetTransactionsCallback received transactions");
 
-                                        for (Transaction transaction : transactions) {
-                                            Optional<byte[]> optRaw = transaction.getRaw();
-                                            if (!optRaw.isPresent()) {
-                                                Log.log(Level.SEVERE, "BRCryptoCWMBtcGetTransactionsCallback completing with missing raw bytes");
-                                                walletManager.getCoreBRCryptoWalletManager().announceGetTransactionsComplete(callbackState, false);
-                                                return;
+                                        try {
+                                            for (Transaction transaction : transactions) {
+                                                Optional<byte[]> optRaw = transaction.getRaw();
+                                                if (!optRaw.isPresent()) {
+                                                    Log.log(Level.SEVERE, "BRCryptoCWMBtcGetTransactionsCallback completing with missing raw bytes");
+                                                    walletManager.getCoreBRCryptoWalletManager().announceGetTransactionsComplete(callbackState, false);
```

Comment:
This can't be here; would result in two calls to `announceGetTransactionsComplete`